cmd/acikubectl/cmd/debug.go

```diff
@@ -109,7 +109,7 @@ func addFileToTarball(path string, tarWriter *tar.Writer) error {
 
 func createTarForClusterReport(tarWriter *tar.Writer) error {
 	// Create tar file out for all kubectl cp files
-	createTarCmd := exec.Command("tar", "-cvf", "hostfiles.tar", "cluster-report/hostfiles/")
+	createTarCmd := exec.Command("tar", "-cvf", "hostfiles.tar", "hostfiles/")
```

Comment:
Good catch. Let's make sure acc-pytests is updated accordingly.

Reply:
I do recollect now why we had it the way it was earlier. By removing "cluster-report" from "cluster-report/hostfiles/" what it results in is that it creates two separate directories when we untar it:
drwxrwxr-x 5 noiro noiro    4096 May 20 00:43 hostfiles
drwxrwxr-x 5 noiro noiro    4096 May 20 00:43 cluster-report

This is not desirable. When we untar we want everything to go into one directory: "cluster-report". We can create whatever directories we want under that, but at top level there should only be one directory.

Reply:
I understand this is undesirable, but I would like to point out 2 things here:
1. The hostfiles dir created here only has the dir tree and no log files.
2. Earlier this hostfiles dir would be created inside cluster-report, but would need to be deleted first before hostfiles.tar could be extracted. hence I made this change.

Could we leave this change as is and I can create a seperate issue to refactor acikubectl to not produce this empty dir at all.

Reply:
Okay thats fine, but note that we have test all over again for every new commit. Can you evaluate the effort to fix it in this patch (you can create a separate issue, address both in this PR).